tests/test_yaml2sbml/ode_input_invalid_SBML_identifier.yaml

```
@@ -0,0 +1,117 @@
+time:
+    variable: t
+
+parametersR:
```

```suggestion
parameters:
```
Did this typo come from `yaml2sbml` or manually? Might be worth checking

---

That is a very good point! Thank you! I am a bit surprised, since apparently this file is never used for testing and also was (looking also at the name) never intended to test these errors. This is a copy-paste error, that I will quickly fix in another PR before we do the new release :)

---

> Did this typo come from `yaml2sbml` or manually? Might be worth checking

And no, fortunately copy paste error from another test case testing exactly these issues :)